hknweb/candidate/admin.py

```diff
@@ -32,12 +50,30 @@ def export_as_csv(self, request, queryset):
 
         writer.writerow(field_names)
         for obj in queryset:
-            row = writer.writerow([getattr(obj, field) for field in field_names])
+            writer.writerow([getattr(obj, field) for field in field_names])
 
         return response
 
     export_as_csv.short_description = "Export selected as csv"
 
+    def csec_confirm(self, request, queryset):
+        for obj in queryset:
+            if obj.csec_confirmed is not True:
+                obj.csec_confirmed = True
+                obj.save()
+                OffChallengeAdmin.check_send_email(request, obj)
+
+    csec_confirm.short_description = "Mark selected as confirmed (csec)"
+
+    def csec_reject(self, request, queryset):
+        for obj in queryset:
+            if obj.csec_confirmed is not False:
+                obj.csec_confirmed = False
+                obj.save()
+                OffChallengeAdmin.check_send_email(request, obj)
```

Comment:
This should be `self.check_send_email`, because even if it's a static method we want this code to survive class renaming.

Reply:
Huh I never heard that

Reply:
Isn't there a thing called refactor?

Reply:
An IDE feature, perhaps, but depending on it for a team with inconsistent editors and IDEs will require more effort than saved.